examples/tutorials/TodoApp/schema.prisma

```diff
@@ -0,0 +1,24 @@
+datasource db {
+  provider = "sqlite"
+  // Wasp requires that the url is set to the DATABASE_URL environment variable.
+  url      = env("DATABASE_URL")
+}
+
+// Wasp requires the `prisma-client-js` generator to be present.
+generator client {
+  provider = "prisma-client-js"
+}
+
+// Define your Prisma schema below
```

Comment:
This comment feels a bit reundantn, since you can also add stuff above if you want, to existing generator or datasource blocks, or whatever really, no reason just to define stuff below.

Reply:
This is the comment each Prisma file has now (since it's a part of the `basic` template). Do you think we should remove it all the places?

Reply:
Yeah I think I would, I think it gives a bit of a wrong idea.

Reply:
Makes sense, I'll then remove this comment everywhere :)